packages/deployment/ansible/roles/cosmos-genesis/files/faucet-helper.sh

```diff
@@ -78,14 +78,14 @@ while [[ ${#rpcAddrs[@]} -gt 0 ]]; do
         msg1='[]'
       fi
       txfile="/tmp/faucet.$$.json"
+      signedtxfile="/tmp/faucet.$$.signed.json"
       trap 'rm -f "$txfile"' EXIT
```

Comment:
We should clean up the signed tx file on exit also.
```suggestion
      trap 'rm -f "$txfile" "$signedtxfile"' EXIT
```